contrib/tools/relay-server/README.md

```diff
@@ -0,0 +1,5 @@
+# Start the `relay-server` server
+
```

Comment:
Could you add a short summary of what this binary does, what port does it open, how it behaves?